Library/Formula/teensy_loader_cli.rb

```diff
@@ -1,17 +1,34 @@
 class TeensyLoaderCli < Formula
   desc "Command-line integration for Teensy USB development boards"
   homepage "https://www.pjrc.com/teensy/loader_cli.html"
-  url "https://www.pjrc.com/teensy/teensy_loader_cli.2.1.zip"
-  sha256 "dafd040d6748b52e0d4a01846d4136f3354ca27ddc36a55ed00d0a0af0902d46"
+  url "https://github.com/PaulStoffregen/teensy_loader_cli/archive/refs/tags/2.2.tar.gz"
+  sha256 "103c691f412d04906c4f46038c234d3e5f78322c1b78ded102df9f900724cd54"
+
+  depends_on "make"
+  depends_on "libusb"
```

Comment:
Looks like it pulls in `libusb-compat` too?

Reply:
You're right, it snuck in as it was already installed.
libusb-compat depends on libusb so I've switched the line over to libusb-compat.